config/rwt.json5

```diff
@@ -0,0 +1,18 @@
+{
+  app: {
+    url: 'https://relay.rif-wallet-services.testnet.rifcomputing.net',
+    port: 8090,
+    devMode: true,
+    logLevel: 0,
+    disableSponsoredTx: false,
+    feePercentage: '0',
```

Comment:
Perhaps this would be a good place to create a task to review the type choice of some of these props. For instance, I'm not convinced that we'd ever need better resolution on percentage values than what type `number` gives us.

Reply:
I had the same doubt indeed, just kept the same type used before.